scripts/Cart.js

```diff
@@ -0,0 +1,104 @@
+import Cartline from "./Cartline"
+
+export default class Cart {
+    constructor() {
+        this.cartlines = []
+        this.numberOfItems = 0
+        this.totalPrice = 0
+    }
+
+    getCartlines() {
+        return this.cartlines
+    }
+    getNumberOfItem() {
+        return this.numberOfItems
+    }
+    getTotalPrice() {
+        this.totalPrice = this.calcTotalPrice()
+        return this.getTotalPrice
+    }
+    setCartlines(cartlines){
+        this.cartlines = cartlines
+    }
+    setNumberOfItems(numberOfItems){
+        this.numberOfItems = numberOfItems
+    }
+
+    calcTotalPrice() {
+        let s = 0
+        this.cartlines.forEach(cl => {
+            s += parseInt(cl.getTotalPrice())
+        })
+        return s
+    }
+
+    isCartlineExist(cartline) {
+        console.log("isCartlineExist, wanted cartline id = " + cartline.getID());
+        
+        for(let i = 0; i< this.cartlines.length; i++) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
will it work instead?